rosdep/python.yaml

```diff
@@ -5409,6 +5409,10 @@ python3-click:
   gentoo: [dev-python/click]
   rhel: ['python%{python3_pkgversion}-click']
   ubuntu: [python3-click]
+python3-collada:
+  debian: [python3-collada]
```

Comment:
It looks like this is only available on bullseye and later.  Thus, to maintain compatibility with our older distributions, I suggest the following:

```
debian:
  '*': [python3-collada]
  buster: null
  stretch: null
```